user-interface/src/lib/components/CaseNumberInput.tsx

```diff
@@ -72,7 +72,7 @@ function CaseNumberInputComponent(props: CaseNumberInputProps, ref: React.Ref<In
 
   return (
     <Input
-      {...props}
+      {...otherProps}
```

Comment:
This had also created an issue becaues we were passing React component attributes down to the actual HTML element.  So the code was complaining that we were assigning mixed case attribute names to an input element)